go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/projectriff/cnab-riff
+
+go 1.12
```

Comment:
Not sure this is a good idea. It would be better to allow versions >=1.12. Maybe the best way is to state the Go version requirement in the README.

Reply:
this was generated by `go mod init`, and I can't seem to find a way to specify `>=` 
When I changed it to say `>=1.12` I get the error `note: module requires Go >=1.12` even though I have `go1.12.6`.